microbench/src/main/java/io/netty/microbench/channel/DefaultChannelPipelineBenchmark.java

```diff
@@ -34,6 +35,7 @@
 
 @Warmup(iterations = 5)
 @Measurement(iterations = 5)
+@Fork(5)
```

Comment:
can you re-run the numbers by adding @CompilerControl(DONT_INLINE) in the `propagateEvent` method?
I have the suspect that there is some weird things going on there...